e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -1698,47 +1698,26 @@ describe("scenarios > dashboard > entity id support", () => {
   });
 
   it("when loading `/dashboard/entity/${non existing entity id}`, it should show a 404 page", () => {
-    const invalidSlug = "x".repeat(21);
-    cy.visit(`/dashboard/entity/${invalidSlug}`);
+    const nonExistingEntityId = "x".repeat(21);
+    cy.visit(`/dashboard/entity/${nonExistingEntityId}`);
 
     H.main().findByText("We're a little lost...").should("be.visible");
   });
 
-  it("when loading `/dashboard/entity/${entity-id}?tab=${non-existing-tab-entity-id}`, it should still load the dashboard correctly", () => {
```

Comment:
This is not supported anymore, we don't support sequential ids/slugs in tabs as it doesn't make sense. If you use this path you have to use entity ids for both the dashboard and the tab